.ci/pytorch/windows/arm64/bootstrap_buildtools.bat

```diff
@@ -18,31 +19,46 @@ curl -L -o "%INSTALLER_FILE%" %DOWNLOAD_URL%
 echo Installing Visual Studio Build Tools with C++ components...
 echo Installing MSVC %MSVC_VERSION%
 if "%MSVC_VERSION%" == "latest" (
-    "%INSTALLER_FILE%" --norestart --nocache --quiet --wait --installPath "%DEPENDENCIES_DIR%\VSBuildTools" ^
+    "%INSTALLER_FILE%" --norestart --quiet --wait --installPath "%DEPENDENCIES_DIR%\VSBuildTools" ^
```

Comment:
When such significant change of the list is done, I 'd sort them by component names alphabetically as well to make the future maintenance easier.

Reply:
@ozanMSFT does it mean that this is still a draft PR or a final version?